Gemfile

```diff
@@ -1,3 +1,10 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :development do
```

Comment:
I think was something mechanical me ... from Rails. I will remove and move to the main group/